src/js/components/ColumnChooser.js

```diff
@@ -3,22 +3,34 @@
 import React from "react"
 
 import {XColumnChooserMenu} from "./ColumnChooserMenu"
+import BrimTooltip from "./BrimTooltip"
+import ColumnsIcon from "../icons/ColumnsIcon"
 import DropMenu from "./DropMenu"
-import MenuBarButton from "./MenuBarButton"
+import ToolbarButton from "./ToolbarButton"
 
 type Props = {}
 
 export default class ColumnChooser extends React.Component<Props> {
   render() {
     return (
-      <div className="column-chooser-wrapper">
+      <div className="columns-button">
         <DropMenu
           position="right-wall"
           value="columns"
           menu={XColumnChooserMenu}
           dim={true}
         >
-          <MenuBarButton>Columns</MenuBarButton>
+          <div
```

Comment:
might be nice to refactor the BrimTooltip wrapper so that it can receive the `data-*` attributes, pass them into and return a div like this one, and then that div would wrap around the actual `<ReactTooltip {passing in the non-data-* related props} />` and `{this.props.children}` so that the new interface would be:
```
<BrimTooltip data-tip="helpful message">
            <ToolbarButton icon={<ColumnsIcon />} />
            <label>Columns</label>
</BrimTooltip>
```
wdyt?

Reply:
I see the appeal. But at the moment, I do think it is convenient to keep the target (the element with the mouseover events) and the tooltip component as two distinct pieces. But maybe more will be revealed and it will be more convenient to do as you suggest here.